.evergreen/atlas_data_lake/config.yml

```diff
@@ -6,7 +6,10 @@ hostRegion: local
 
 # Disable billing to silence a "no billing configured" warning.
 billing:
-  enabled: false
+  billing:
```

Comment:
there's 2 `billing`: lines here, the second one should be removed. otherwise lgtm

Reply:
Fixed, pushed that just as I sent you the link 😅 TY

Reply:
ha all good, looks correct now!